Dockerfile

```diff
@@ -12,15 +12,16 @@ RUN go mod download
 # Copy the go source
 COPY . /workspace
 
-# Build with make to apply all build logic difined in Mekefile
+# Build with make to apply all build logic defined in Makefile
 RUN make build
+# Build host-local cni
+RUN git clone https://github.com/containernetworking/plugins.git ; cd plugins ; git checkout v1.2.0 -b v1.2.0
+RUN cd plugins ; go build -o plugins/bin/host-local ./plugins/ipam/host-local
 
 # Use distroless as minimal base image to package the manager binary
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM gcr.io/distroless/static:nonroot
+FROM gcr.io/distroless/base-debian11:latest
```

Comment:
wasnt working with prev image

Reply:
as I mention in https://github.com/Mellanox/nvidia-k8s-ipam/pull/4#discussion_r1183074220. The noroot image don't allow to copy host-local cni to host location.